Pre_processing_scripts/01_Sort_IT_Extracts.R

```diff
@@ -0,0 +1,129 @@
+rm(list = ls())
+devtools::load_all()
+
+it_extract_path <- file.path(
+  get_slf_dir(),
+  "IT_extracts"
+)
+it_extract_anon_path <- file.path(
+  it_extract_path,
+  "anon-chi-IT"
+)
+
+is_chi_in_file <- function(filename) {
+  data <- read.csv(filename, nrow = 1)
+  return(grepl("UPI", names(data)) %>% any())
+}
```

Comment:
I wonder if we can split some of the functions out into a separate functions script and use source to call them. Some of these can be applied to sort_BI_extracts and write_anon_chi_files

Reply:
Yes. That's right. Put this into a new issue and get it done in the next update.